### PR TITLE
Add instructions on how to run scrapbook locally

### PR DESCRIPTION
**Pull request by jessicard**
Add instructions on how to run the project locally.

Also adds support for imgur.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,21 @@ This site exposes a public JSON API powered by [Next.js API routes](https://next
 - [`/api/r/:emoji`](https://scrapbook.hackclub.com/api/r/hardware) – Get all posts tagged with a specific emoji
 - [`/:username.png`](https://scrapbook.hackclub.com/sampoder.png) – Get a user's avatar as an image URL
 
+## Running Locally
+
+1. Clone this repository
+   - `git clone git@github.com:hackclub/scrapbook.git`
```
Comment (maxwofford) [inline README.md:99]:
```suggestion
   - `git clone git@github.com:hackclub/scrapbook.git; cd scrapbook`
```

File: README.md [hunk 2]
```diff
@@ -93,6 +93,22 @@ This site exposes a public JSON API powered by [Next.js API routes](https://next
 - [`/api/r/:emoji`](https://scrapbook.hackclub.com/api/r/hardware) – Get all posts tagged with a specific emoji
 - [`/:username.png`](https://scrapbook.hackclub.com/sampoder.png) – Get a user's avatar as an image URL
 
+## Running Locally
+
+1. Clone this repository
+   - `git clone git@github.com:hackclub/scrapbook.git; cd scrapbook`
```
Comment (exu3) [inline README.md:99]:
```suggestion
   - `git clone https://github.com/hackclub/scrapbook.git && cd scrapbook`
```
If a user doesn't have their SSH key on GitHub, they would have to add that before doing this step; would it just be easier to change the instructions to clone using HTTPS?

Screen Shot 2022-03-02 at 10 24 01

File: README.md [hunk 3]
```diff
@@ -93,6 +93,22 @@ This site exposes a public JSON API powered by [Next.js API routes](https://next
...
+1. Install dependencies
+   - `yarn`
+1. Link Vercel to your account for deployment
+   - `yarn vercel`
+   - _Note_: The project name in Vercel is `summer-streaks`
+1. Pull environment variables from Vercel
+   - `yarn vercel env pull`
```
Comment (exu3) [inline README.md:106]:
```suggestion
   - `yarn vercel pull`
```

Screen Shot 2022-03-02 at 10 21 44

**Review by exu3 — COMMENTED** [review]
Thanks for adding these instructions; they're super helpful!

File: README.md (hunk 2 above)
Comment (jessicard) [reply README.md]: @exu3 Smart! Done in [e80b1bb](https://github.com/hackclub/scrapbook/pull/280/commits/e80b1bb18329133ad27406a588bc5822aad3ded4)

File: README.md (hunk 3 above)
Comment (jessicard) [reply README.md]: @exu3 Nice, I didn't notice that 🔍  Fixed in [4477ed4](https://github.com/hackclub/scrapbook/pull/280/commits/4477ed44ccfdfad78ac6b2c0fb52730d67d7110a)

File: README.md (hunk 1 above)
Comment (jessicard) [reply README.md]: Good call! Fixed in [c21bab9](https://github.com/hackclub/scrapbook/pull/280/commits/c21bab97ca9f15305be40debb8d45fdcf543c85b)

**Pull request merged**